flytekit/extend/backend/base_agent.py

```diff
@@ -215,7 +216,9 @@ async def _get(self, resource_meta: bytes) -> GetTaskResponse:
             else:
                 res = self._agent.get(grpc_ctx, resource_meta)
             state = res.resource.state
-            logger.info(f"Task state: {state}, State message: {res.resource.message}")
+            print(f"{self._entity.name} task state: {State.Name(int(state))}, message: {res.resource.message or None}")
```

Comment:
here too

Reply:
you probably want this in the agent executor mixin?

Reply:
it is in the mixin.